pkg/disttask/framework/scheduler/interface.go

```diff
@@ -125,8 +129,15 @@ type Extension interface {
 	GetNextStep(task *proto.Task) proto.Step
 }
 
+// Param is used to pass parameters when creating scheduler.
+type Param struct {
```

Comment:
Maybe rename to a more meaningful name. But I have no idea now... Wait for other reviewers comment..

Reply:
SchedulerParam is better, but as https://go.dev/blog/package-names stated:

> Avoid repetition. Since client code uses the package name as a prefix when referring to the package contents, the names for those contents need not repeat the package name. The HTTP server provided by the http package is called Server, not HTTPServer. Client code refers to this type as http.Server, so there is no ambiguity.